Chapters/4-2-Chapter24.md

```diff
@@ -0,0 +1,211 @@
+
+"Oi, let's head toward the wall to the left," she said, "This place is big."
+
+We turned and paddled softly, but it didn't take long for us to hit ground. The boat slid ashore onto a sand-and-pebble jetty with a loud _shhhh-ch_, and stopped abruptly. Pomri wasted no time untying the rope and hopping out.
+
+"Let's go," she said, already off the boat, and I instinctively clambered to the front, trying to keep my balance. I could tell she was agitated, that she felt vulnerable in a place like this.
+
+"Aye. I'm coming," I said.
+
+Before I could set foot on the ground, I heard movement: footsteps on pebbles echoing from every wall. I stopped in my tracks to listen. They weren't deliberate and quick like a bandit or an assassin – more uncontrolled, and barely hanging on. Pomri was equally startled and moved closer to hold me back. With a swift gesture, she held her hand out in front of my mouth mouth, a command to stay quiet.
+
+At the edge of the outside light, I saw two giant silhouettes of saffas, perched at the mouth of the cove and looking in just out of the light. But somewhere between the saffas and us, that cadence of slow footsteps rattled off the walls. If it were just a throngia, we could push the boat off and paddle away. But a saffa can swim and run much faster than we could ever hope to paddle or run.
+
+Pomri began to approach, to put distance between me and the walking sound. She was deliberately slow and silent. Only taking a step with each wave that boomed into the mouth.
+
+But then the footsteps stopped with a larger thud, as if the mysterious person had plopped all their body weight on the ground. _We are intruders here._
+
+Once she was close enough, Pomri spoke up, "We don't mean any-," but before she could finish, the person in the dark had found Pomri and held a long wooden pole to her chest. She took a step back and tried to force the pole away, but it had disappeared into the darkness as the shadow fled back toward the saffas. I ran after, evading Pomri's hands and protests not to go. _This isn't an enemy, It can't be._
+
+The stranger passed between the saffas, and out of the cove where the remaining Ara-light shone against hae back. I was aghast at hae feeble figure, hydroderm swollen with cold and sunken with hunger. But before I could reach hae, one of the saffas had turned and picked hae up, while the other stood ground guarding their getaway. Through the growling and shuffling of feet, I heard a sound I'd not expected to hear: the ringing jingles of a binding pendant as it rocked this way and that around the saffa's neck. And then I saw him – the thick fur and the big ears and the myriad different mannerisms that were unmistakable for anyone else but my best friend.
+
+"Laen! You're alive!"
```

Comment:
yaya!!!! this is the happiest moment of the whole book! Laen Laen is not smitterrens. 🐴👧